### PR TITLE
Introduce Charlotte Recommendation 

### LINKED ISSUE #20: Add recommendation file for Charlotte, NC
- [ ] create a branch
- [ ] add a commit
- [ ] Open a Pull request
- [ ] Review
- [ ] Merge

### PR DESCRIPTION
**Pull request by sharmadimple2510**
closes #20

### CODE REVIEW

**Comment by HyimHK** [comment]
👍

File: dimple-Charlotte.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Charlotte
+GitHub for Developers Training
+
+### Places to eat
```
Comment (HyimHK) [inline dimple-Charlotte.md:4]: Do you recommend any 👍 🍣 restaurants in Charlotte

Comment (sharmadimple2510) [reply dimple-Charlotte.md:4]: You can find many in Charlotte downtown

**Review by HyimHK — APPROVED** [review]
LGTM

**Pull request merged**